test/pipeline_test.dart

```diff
@@ -0,0 +1,113 @@
+// Copyright (c) 2017, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:http/http.dart';
+import 'package:test/test.dart';
```

Comment:
Yes; see [the style guide](https://www.dartlang.org/guides/language/effective-dart/style#ordering).